snews_cs/cs_email.py

```diff
@@ -24,26 +37,46 @@ def send_email(alert_content):
     for k, v in alert_content.items():
         space = 40 - len(k)
         pretty_alert += f'{k} :{r" "*space}{v}\n'
+    subject = "SNEWS COINCIDENCE" + datetime.utcnow().isoformat()
```

Comment:
```
subject = "SNEWS COINCIDENCE" + datetime.utcnow().isoformat()
```
can be 
```
subject = f"SNEWS COINCIDENCE {datetime.utcnow().isoformat()}"
```
the main change is a whitespace between the text and the date

Reply:
Change committed.